gooddata-fdw/tests/execute/test_execute_compute_table.py

```diff
@@ -2,17 +2,25 @@
 from pathlib import Path
 
 import vcr
+from tests_support.vcrpy_utils import VCR_MATCH_ON, CustomSerializerYaml, custom_before_request, custom_before_response
 
 from gooddata_fdw import GoodDataForeignDataWrapper
-from tests import VCR_MATCH_ON
 
 _current_dir = Path(__file__).parent.absolute()
 _fixtures_dir = _current_dir / "fixtures"
 
-gd_vcr = vcr.VCR(filter_headers=["authorization", "user-agent"], serializer="json", match_on=VCR_MATCH_ON)
+gd_vcr = vcr.VCR(
```

Comment:
Create procedure which builds vcrpy with custom configuration.

Reply:
Done. Procedure created in tests-support package.